turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -667,9 +698,17 @@ impl ResolveResult {
                             request,
                             match item {
                                 ResolveResultItem::Source(source) => asset_fn(source).await?,
-                                ResolveResultItem::External(s, ty) => {
-                                    ModuleResolveResultItem::External(s, ty)
-                                }
+                                ResolveResultItem::External {
+                                    name,
+                                    ty,
+                                    // TODO remove this whole function? it's easy to drop traced
+                                    // externals now
```

Comment:
What's up with this?

Reply:
I've turned this case into a `bail!` in the next PR.